tests/game_modules/test_rules.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from python_tictactoe.game_modules.rules import Rules
+from python_tictactoe.game_modules.board import Board
+
+class TestRules(object):
+
+    def test_all_elements_are_the_same(self):
+        test_board = Board()
```

Comment:
test_board and test_rules duplication in these tests to be extracted into a fixture